test/runnable/test52.d

```diff
@@ -1,4 +1,11 @@
 // PERMUTE_ARGS:
+/*
+RUN_OUTPUT:
+---
+this()
+~this()
```

Comment:
CI not happy with this (missing the printf from module dtor, although why you'd do that is beyond me.

Reply:
It's checking that the module dtor is **not** run because X.~this should've aborted the test.